native/acter/src/api/spaces.rs

```diff
@@ -82,182 +109,245 @@ impl Space {
         is_acter_space(&self.inner).await
     }
 
-    pub(crate) async fn add_handlers(&self) {
+    pub(crate) async fn add_handlers(&mut self) {
         self.room
             .client()
             .add_event_handler_context(self.client.executor().clone());
         tracing::trace!(room_id=?self.room.room_id(), "adding handlers");
-        let room_id = self.room_id().to_owned();
         // FIXME: combine into one handler
 
         // Tasks
-        self.room.add_event_handler(
-            move |ev: SyncTaskListEvent,
-                  client: SdkClient,
-                  //  room: Room,
-                  encryption_info: Option<EncryptionInfo>,
-                  Ctx(executor): Ctx<Executor>| {
-                let room_id = room_id.clone();
-                async move {
+        let handle =
```

Comment:
yeah, I still don't have any explanation on why we have to store room handlers now ... Care to give one?

I fear that this isn't a proper way to do it, as we don't necessarily keep those spaces around forever either - we pull them in and destroy them and pull them in again at a whim... if we need to store these event-handlers for longevity, this isn't the place ...